### PR TITLE
html-forms exercise - Jay Ghil

### PR DESCRIPTION
**Pull request by jghil**
html-forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <select name="reasons" id="reasons-options">
+          <option value="Select an option">Select an option</option>
+          <option value="Because it's interesting">Because it's interesting</option>
+          <option value="Because it's fun">Because it's fun</option>
+          <option value="Because it's powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <p>Favorite Language (choose one)</p>
+      <div>
+        <input type="radio" name="language" value="HTML" /> HTML
```
Comment (BrettAlbright) [inline html-forms/index.html]: make sure these inputs have a label with the language text inside, you can wrap the inputs with the label tags if desired.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looking great Jay, make sure to add the labels for the radio inputs

File: html-forms/index.html (hunk 1 above)
Comment (jghil) [reply html-forms/index.html]: Will do!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**